great_expectations/types/__init__.py

```diff
@@ -65,7 +65,7 @@ def __contains__(self, key):
     def __len__(self):
         return len(self.__dict__)
 
-    def keys(self) -> set:
+    def keys(self):
         return self.to_dict().keys()
```

Comment:
@cdkini I got rid of the type hint -- we either apply type hints to all methods of the class or none (the interface in `Python` does not use type hinting, which of course does not mean we should not.  We used to have a type hint that was generating a PyCharm warning, because it conflicted with the type returned by `self.to_dict().keys()`.  Thanks.